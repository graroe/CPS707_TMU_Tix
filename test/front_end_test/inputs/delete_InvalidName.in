login
admin
delete
NotAnEvent
!q
logout