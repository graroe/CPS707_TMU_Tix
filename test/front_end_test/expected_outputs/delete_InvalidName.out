Enter session type <sales or admin>: 
Admin login successful.
Enter event name: 
Event not found. 
Enter event name: 
Transaction canceled
Session terminated.